### PR TITLE
Add @(), ceiling(), and translate() functions

### LINKED ISSUE #774: Document `translate`(), `@`(), and `ceiling`()
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/8998

### Which part(s) of the docs might be affected or should be updated? And how?

- Update abs and ceil in https://www.risingwave.dev/docs/upcoming/sql-function-mathematical/#mathematical-functions
- Add translate() to https://www.risingwave.dev/docs/upcoming/sql-function-string/

### Reference

`translate`():

- Postgres: https://www.postgresql.org/docs/15/functions-string.html
- Tutorial: https://www.postgresqltutorial.com/postgresql-string-functions/postgresql-translate/

### PR DESCRIPTION
**Pull request by bernscode**
Add @(), ceiling(), and translate() functions.

## Info
- **Description**: 
Add @(), ceiling(), and translate() functions

- **Preview**: 
[ Paste the preview link to the edited page here. ]

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/8998

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/774

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/sql/functions-operators/sql-function-mathematical.md [hunk 1]
```diff
@@ -23,13 +23,14 @@ title: Mathematical functions and operators
 
 | Function | Description | Example |
 | ----------- | ----------- | ----------- | 
-| abs ( *input_value* ) → *absolute_value* | Returns the absolute value of *input_value*. The *input_value* can be type int or decimal. The return type is the same as the *input_value* type. | abs(-3) → 3 |
-| ceil ( *numeric_input* ) → *integer_output* <br /> ceil ( *double_precision_input* ) → *integer_output* | Returns the nearest integer greater than or equal to the argument. | ceil(1.23559) → 2 <br /> ceil(-1.23559) → -1 |
+| abs ( *input_value* ) → *absolute_value* <br /> @ ( *input_value* ) → *absolute_value* | Returns the absolute value of *input_value*. The *input_value* can be type int or decimal. The return type is the same as the *input_value* type. | abs(-3) → 3 <br /> @(-3) → 3 |
```
Comment (xiangjinwu) [inline docs/sql/functions-operators/sql-function-mathematical.md:26]: FYI the parentheses after `@` is optional. `@ -3` or `@ column1` is completely valid.

File: docs/sql/functions-operators/sql-function-mathematical.md [hunk 2]
```diff
@@ -23,13 +23,14 @@ title: Mathematical functions and operators
...
+| ceil ( *numeric_input* ) → *integer_output* <br /> ceiling ( *double_precision_input* ) → *integer_output* | Returns the nearest integer greater than or equal to the argument. ciel() and ceiling() function the same. | ceil(1.23559) → 2 <br /> ceiling(-1.23559) → -1 |
```
Comment (xiangjinwu) [inline docs/sql/functions-operators/sql-function-mathematical.md]: Maybe we can use the shorter `ceil` consistently in the leftmost column, and just mention `ceiling` is accepted as an alias in description. The example can still show usage of both. This avoid the interpretation of *ceil for numeric and ceiling for double precision*.

btw typo `ciel` -> `ceil`

**Review by xiangjinwu — APPROVED** [review]

File: docs/sql/functions-operators/sql-function-mathematical.md (hunk 2 above)
Comment (bernscode) [reply docs/sql/functions-operators/sql-function-mathematical.md]: @xiangjinwu Good point! I've made the update on the latest commit. Thanks!

**Review by hengm3467 — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**